drivers/modem/Kconfig.gsm

```diff
@@ -0,0 +1,29 @@
+config MODEM_GSM_PPP
+	bool "Support GSM modems"
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	help
+	  Enable GSM modems that support standard AT commands and PPP.
+
+if MODEM_GSM_PPP
+
+config MODEM_GSM_UART_NAME
+       string "UART device name the modem is connected to"
+
+config MODEM_GSM_INIT_PRIORITY
+       int "Init priority for the GSM modem driver"
+       default 42
+       range 0 99
+       help
+         The GSM modem is initialized in POST_KERNEL using priority in
+	 the range 0-99.
+
+config MODEM_GSM_APN
+       string "Access Point Name"
+       default "internet"
+       help
+         Specify Access Point Name, i.e. the name to identify Internet or
+	 IP GPRS/3g/4G/etc. cellular data context.
+
+endif
```

Comment:
you'll need an line break here or make menuconfig will blow out

Reply:
Fixed.